bind/motor-mobile/bind.go

```diff
@@ -19,7 +19,6 @@ type MotorCallback interface {
 var (
 	objectBuilders map[string]*object.ObjectBuilder
 	instance       mtr.MotorNode
-	callback       ct.MotorCallback
```

Comment:
Why is this being removed?

Reply:
It doesnt need to be there since its being passed to the Motor Node